webapp/packages/core-connections/src/DatabaseAuthDialog/DatabaseAuthDialog.tsx

```diff
@@ -93,15 +73,19 @@ export const DatabaseAuthDialog = observer(function DatabaseAuthDialog({
   const isAuthNeeded = connection.connectionInfo?.authNeeded;
   const isSSHAuthNeeded = sshConfig?.enabled && !sshConfig.savePassword;
 
-  return styled(useStyles(styles))(
+  const databaseAuthenticationTitle = translate('connections_database_authentication');
```

Comment:
please try to simplify variable names, in that case, we have only one title, so you can name it `title`